spec/emerald/parser_spec.rb

```diff
@@ -16,10 +16,17 @@
     end
 
     it "parses two atoms" do
-      source = "hello      world"
+      source = "hello      World"
       ast = Emerald::Parser.new(source).parse
       expect(ast).to eq([Emerald::Atom.new("hello"),
-                         Emerald::Atom.new("world")])
+                         Emerald::Atom.new("World")])
+    end
+
+    it "parses two atoms: a string and an integer" do
```

Comment:
The first one is an atom, not a string. Atoms are just names.

The second is a number, which isn't an atom :)